### DISCUSSION TITLE
Round up percentage

### DISCUSSION THREAD

**Discussion by dhcvn** [nmalkin/plot-likert#45] [General]
Hi, 
Do you consider to add number of decimal percentage option to the percentage function?

**Comment by nmalkin** [comment]
Hi!

Here's an example for how to do that using the current features:
https://gist.github.com/nmalkin/9a31437d3be18d637d0b63e54926c491

At a minimum, I should document this in the user guide. If there's enough interest, I could potentially add a parameter to the plotting function, as you suggested.

**Comment by nmalkin** [comment]
@dhcvn [said](https://gist.github.com/nmalkin/9a31437d3be18d637d0b63e54926c491?permalink_comment_id=4747886#gistcomment-4747886):
> Thank you for your prompt response. I have another question. How can I change size of the label?

This is going to a bit trickier. The way to change the size of the label is to pass an argument like [`fontsize=16`](https://matplotlib.org/stable/api/text_api.html) to the [`bar_label` function](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.bar_label.html) here:

https://github.com/nmalkin/plot-likert/blob/ce15502261b0a41aef9a37c8f97a89baaa2bec17/plot_likert/plot_likert.py#L212-L219

Unfortunately, that's not currently exposed. The two options I can imagine are to:

1. Rummage through the Axis object until you find the bar labels and call [`set_fontsize`](https://matplotlib.org/stable/api/text_api.html#matplotlib.text.Text.set_fontsize) on them
2. Have a local copy/fork of the _plot-likert_ library (it's only a few files) that you modify to your needs

**Reply by dhcvn** [reply]
I think that changing the source file in the local computer is not the optimal options. But It's work for me now. Thank you.

**Reply by nmalkin** [reply]
Yeah, I totally agree, it's obnoxious to have to do. One solution might be to add a new argument to pass through to the `bar_label` function. Someone actually created a fork of this library to do the same thing for customizing the legend: 533b570. I don't love having a million different arguments to `plot_likert`, but I'm open to it if there's enough interest and no better solution.
